CHANGELOG-UNRELEASED.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Renames link tags to link_type. Adds new link tag which can be any string. This is available in validation of links and links can be retrieved based on their tag+type, just tag, just type or retrieve all.  [#1402](https://github.com/holochain/holochain-rust/pull/1402).
```

Comment:
I think this change also needs a "breaking change" note re `hdk::link_entries` and `hdk::get_links`  needing extra params no?